java/core/src/main/java/com/lancedb/lancedb/Database.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.lancedb.lancedb;
+
+import io.questdb.jar.jni.JarJniLoader;
+import java.util.List;
+ 
+/**
+ * Represents LanceDB database.
+*/
+public class Database {
```

Comment:
Lets make the naming conventions be consistent with the rest of the rust/python/nodejs packages.

YOu can take a look of https://docs.rs/lancedb/latest/lancedb/